packages/enzyme-adapter-react-16/src/ReactSixteenAdapter.js

```diff
@@ -617,6 +617,14 @@ class ReactSixteenAdapter extends EnzymeAdapter {
     return isElement(element);
   }
 
+  shouldUpdateComponent(prevProps, root) {
+    if (root.instance() !== null) {
```

Comment:
if there's no instance, shouldn't this function always return `true`?

Reply:
added an SFC test, just to be sure.

Reply:
ya youre right.